content/articles/getting-started-with-python-plug-ins-for-gimp/index.md

```diff
@@ -0,0 +1,158 @@
+You can likewise get GIMP-python documentation [here](http://www.gimp.org/docs/python/)
+
+### Enrolling your content
+On the Windows stage, Python records go in your client's home GIMP folder. For my situation its 
+'C:\Users\KENNEDY\AppData\Roaming\GIMP\2.10\plug-ins' the place where I have saved my `.py` record 
+- Note that framework modules are a hidden folder; thus, to get to Gimp 2.10 modules index in Windows 10 open File Explorer and enter the above path from your GIMP. 
+You can get the above path by opening your GIMP, tapping on 'edit>preferences>folders>plug-ins' 
+The figure beneath shows the way of getting the document area 
+
+![Plug-ins folder](plugins.png) 
+
+the chose path above is for '`system plug-ins`', and the other is for 'user plug-ins' duplicate the framework modules and paste it on your file explorer pathway. Make another text document in the open folder and save it as '.py' 
+On Mac and Linux frameworks, that folder is '~/.gimp-2.6/modules'. 
+The Python script records should in like manner be executable and have the Python translator on the primary line, like the standard substance declarations under.
+There are 12 arguements to the register work you might found them clarified [here](http://www.expmedia.com/content/broadening gimp-python-python-fu-modules section 2). You are needed to enlist your python script all together GIMP to add the modules in one of the GIMP menus. The following is a model
+
+``` python
+#!/usr/bin/python
+From gimpfu import * # same for every plug-ins
+def gimp_plugins(image, drawable):
+    return
+register(
+# name of the order you invoke from the command prompt
+"python_fu_resize",
+# The procedure browser displays information about the plug-in.
+"Saves the image at a maximum width and height",
+# The designer of the plug-in
+"Kennedy Mwangi",
+# The plug-copyright in's holder. 
+"Kennedy Mwangi",
+# The copyright expiration date	 
+"2021",
+# The menu label that the plug-in uses. 
+"<Image>/Image/Resize to max ",
+# The methodology parameters for the plug-in
+[
+    (PF_FILE, "infile", "Path for input file", ""), 
+     (PF_DIRNAME, "save-path", "Path for output filename", ""), 
+],
+# The results of the plug-in’s method
+[
+    menu="<image>/Resize to max../"
+],
+gimp_plugins)
+main()
+```
+The register() system gives any data about your module. It has a few boundaries that let GIMP know how to show menu choices for the modules and what python technique to call when you start your modules 
+There is more data about the register technique's boundaries on the python-Fu console, 
+- on windows explore through 'filters>python-Fu>Console' 
+in any case, in Mac, the executable is at 
+- '/Applications/GIMP.app/Contents/MacOS/python' 
+with libraries at: 
+- '/Applications/GIMP.app/Contents/Resources/lib/python2.7' 
+then, at that point, compose the accompanying order

```

Comment:
You can use ` `` ` for keywords, variables, functions, filenames, image names or directories.

Do this change throughout the article